packages/bridge/src/runtime/composables.ts

```diff
@@ -16,7 +17,7 @@ export const useRuntimeConfig = () => {
   if (!nuxtApp.$config) {
     nuxtApp.$config = reactive(nuxtApp.nuxt2Context.app.$config)
   }
-  return nuxtApp.$config
+  return nuxtApp.$config as RuntimeConfig
```

Comment:
```suggestion
  return nuxtApp.$config as PublicRuntimeConfig
```
At least to the added docs, only the public config is available on the client side

Reply:
`RuntimeConfig` is a merged type with private config marked as optional (as this composable also runs on server side)